src/components/Square.js

```diff
@@ -4,11 +4,15 @@ import PropTypes from 'prop-types';
 import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
+  // For Wave 1 enable this
+  //  Component to alert a parent
   //  component when it's clicked on.
 
-  return <button className='square'>{props.value}</button>;
+  return (
+    <button onClick={() => props.onClickCallback(props.id)} className="square">
```

Comment:
Nice inline handler to call our click callback when the button was clicked. Notice that we can't simply use props.onClickCallback directly as the onClick handler, since we need to pass the id of the clicked square to the callback. If we had used it as the onClick directly, then the event details provided by the browser would have been passed in as the first parameter rather than the id.